integration_test/third_party_apps_data/applications/rabbitmq/debian_ubuntu/install

```diff
@@ -14,12 +14,14 @@ curl -s https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc | \
 
 source /etc/os-release
 case $VERSION_ID in
+  # ubuntu
   20.04)
-    # ubuntu
     echo "deb https://packages.erlang-solutions.com/ubuntu focal contrib" | sudo tee /etc/apt/sources.list.d/rabbitmq.list
     ;;
+  # debian
   10)
-    # debian
+    ;&
+  11)
     echo "deb https://packages.erlang-solutions.com/ubuntu bionic contrib" | sudo tee /etc/apt/sources.list.d/rabbitmq.list
```

Comment:
We removed debian11 from presubmit test last night due to the red tests. Thus the PR tests doesn't catch "This repo url doesn't work for bullseye"..
To better test the debian11 3p tests failures, @dehaansa do you mind following [this instruction](https://github.com/GoogleCloudPlatform/ops-agent/blob/706ee06a61330e1da49f37cce68c9dfdb189a818/integration_test/README.md#setup) here to run command to test the fixes on debian11:
```
PROJECT=xxx TRANSFERS_BUCKET=xxx GOOGLE_APPLICATION_CREDENTIALS="${HOME}/credentials.json" ZONE=us-central1-b PLATFORMS=debian-11 SCRIPTS_DIR=third_party_apps_data go test -v third_party_apps_test.go  -test.parallel=1000  -tags=integration_test  -timeout=4h
```